lms/djangoapps/shoppingcart/tests/test_reports.py

```diff
@@ -84,7 +84,7 @@ def setUp(self):
 
         self.cart = Order.get_cart_for_user(self.second_refund_user)
         CertificateItem.add_to_order(self.cart, self.course_key, self.cost, 'verified')
-        self.cart.purchase(self.second_refund_user, self.course_key)
+        self.cart.purchase(self.second_refund_user.username, self.course_key)
```

Comment:
In purchase method `self.second_refund_user` which is of type `User` assigned to `bill_to_first` which is of type `CharField`, Django 1.8 will raise exception upon saving the model instance.